ha/setup/ha_setup.py

```diff
@@ -523,9 +524,8 @@ def process(self):
                 if node != node_name:
                     Log.info(f"Adding node {node} to Cluster {cluster_name}")
                     self._add_node(node, cluster_user, cluster_secret)
+                    self._configure_stonith(all_nodes_stonith_config, node, enable_stonith)
```

Comment:
I see code at line #522 which will try to configure the stonith from individual nodes.

Reply:
yes and below code at line #527 is for other nodes we are trying to add and not in nodelist.
before adding stonith resource we checking that resources are already added or not in configure_stonith function.